packages/dds/tree/src/events/emitter.ts

```diff
@@ -63,7 +63,7 @@ export interface HasListeners<TListeners extends Listeners<TListeners>> {
 /**
  * Provides an API for subscribing to and listening to events.
  *
- * @remarks Classes wishing to emit events may either extend this class or compose over it.
+ * @remarks Classes wishing to emit events may either extend this class, compose over it, or expose it as a property of type {@link Listenable}.
```

Comment:
Do you think it would be best to nudge readers towards the last option? This comment makes it seem as though they're equally good, which might lead people to make suboptimal choices.

Reply:
Good question. For the moment, I'd like to keep EventEmitter unopinionated. I'm not sure if we're quite ready to say that at the library level, this is always the preferred pattern (since this library is going to be lifted out of ST soon). Furthermore, I'm not sure this is the place to document the preference (e.g. maybe EventEmitter stays unopinionated, but we have docs elsewhere that note the preference for a particular pattern).